cmd/evm/main.go

```diff
@@ -194,6 +193,12 @@ func NewEnv(state *state.StateDB, transactor common.Address, value *big.Int, cfg
 	return env
 }
 
+// ruleSet implements vm.RuleSet and will always default to true
```

Comment:
It would probably be clearer if you wrote "it will always default to Homestead", since once we add some more params, "true" will be mostly meaningless.